tornado-api/src/main/java/uk/ac/manchester/tornado/api/profiler/ChromeEventJSonWriter.java

```diff
@@ -0,0 +1,80 @@
+package uk.ac.manchester.tornado.api.profiler;
+
+import sun.management.VMManagement;
+
+import java.io.File;
+import java.lang.management.ManagementFactory;
+import java.lang.management.RuntimeMXBean;
+import java.lang.reflect.Field;
+import java.lang.reflect.InvocationTargetException;
+
+public class ChromeEventJSonWriter extends JSonWriter<ChromeEventJSonWriter> {
+    static private int pid = 0;
+    ContentWriter NO_ARGS = null;
+
+    ChromeEventJSonWriter() {
+        super();
+        // Java 9 we could use long pid = ProcessHandle.current().pid();
+        // For java 8 we must go through some hoops.
+        try {
+            RuntimeMXBean runtime = ManagementFactory.getRuntimeMXBean();
```

Comment:
When compiling with Corretto-11 I get the following error:

```bash
Compilation failure: 
[ERROR] /tmp/tornadovm-installer/TornadoVM-Amazon-Corretto11/TornadoVM/tornado-api/src/main/java/uk/ac/manchester/tornado/api/profiler/ChromeEventJSonWriter.java:[3,10] error: package sun.management is not visible
[ERROR]   (package sun.management is declared in module java.management, but module tornado.api does not read it)
[ERROR] /tmp/tornadovm-installer/TornadoVM-Amazon-Corretto11/TornadoVM/tornado-api/src/main/java/uk/ac/manchester/tornado/api/profiler/ChromeEventJSonWriter.java:[6,16] error: package java.lang.management is not visible
[ERROR]   (package java.lang.management is declared in module java.management, but module tornado.api does not read it)
[ERROR] /tmp/tornadovm-installer/TornadoVM-Amazon-Corretto11/TornadoVM/tornado-api/src/main/java/uk/ac/manchester/tornado/api/profiler/ChromeEventJSonWriter.java:[7,16] error: package java.lang.management is not visible
[ERROR]   (package java.lang.management is declared in module java.management, but module tornado.api does not read it)
``` 

Is this code also valid using Java >= 8

Reply:
I will pull this bean code.  Will just use 0 for pid.